src/ext/java/org/opentripplanner/ext/vectortiles/layers/stops/DigitransitRealtimeStopPropertyMapper.java

```diff
@@ -0,0 +1,59 @@
+package org.opentripplanner.ext.vectortiles.layers.stops;
+
+import static org.opentripplanner.ext.vectortiles.layers.stops.DigitransitStopPropertyMapper.getRoutes;
+import static org.opentripplanner.ext.vectortiles.layers.stops.DigitransitStopPropertyMapper.getType;
+
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.util.Collection;
+import java.util.List;
+import java.util.Locale;
+import org.opentripplanner.apis.support.mapping.PropertyMapper;
+import org.opentripplanner.framework.i18n.I18NStringMapper;
+import org.opentripplanner.inspector.vector.KeyValue;
+import org.opentripplanner.routing.alertpatch.AlertEffect;
+import org.opentripplanner.transit.model.site.RegularStop;
+import org.opentripplanner.transit.service.TransitService;
+
+public class DigitransitRealtimeStopPropertyMapper extends PropertyMapper<RegularStop> {
+
+  private final TransitService transitService;
+  private final I18NStringMapper i18NStringMapper;
+
+  public DigitransitRealtimeStopPropertyMapper(TransitService transitService, Locale locale) {
+    this.transitService = transitService;
+    this.i18NStringMapper = new I18NStringMapper(locale);
+  }
+
+  @Override
+  protected Collection<KeyValue> map(RegularStop stop) {
+    Instant currentTime = ZonedDateTime.now(transitService.getTimeZone()).toInstant();
```

Comment:
```suggestion
    Instant currentTime = Instant.now();
```

Now need for the detour via the ZonedDateTime.

Reply:
Really? I thought the time zone might be important in this context to ensure the alert is actually valid according to local time.

Reply:
If you call toInstant all timezone information is lost again. An instant is just a number of seconds since 1970-01-01 UTC.

Reply:
And since we only want to give an answer at the time of someone asking and not the future/past the time zone doesn't matter.

Reply:
Haha, you're right. Somehow tricked myself into thinking the instant is different in different places. 😂 Thanks!